### DISCUSSION TITLE
Convert it to typescript? :)

### DISCUSSION THREAD

**Discussion by mauriciocarnieletto** [chishui/JSSoup#26] [Ideas]
I've been using this module for a while now. (I love BeutifulSoup as well), and I'm thinking about "translate" it to typescript.
I've already created a fork from this project and I'm starting it. 
Once I'm finished, can I make a pull request?

Tks :)

**Comment by chishui** [comment]
Hey, may I understand what's the benefits of converting it to TypeScript? 
Would this help https://github.com/chishui/JSSoup/issues/20#issuecomment-860755328?
